docs/tutorials/setup/firstprogram.mdx

```diff
@@ -33,11 +33,8 @@ Make sure to save it (or change the vscode `files.autoSave` setting to
 
 Spaces are important in Toit, so make sure that the `print` line is indented
 compared to the `main:` line. Generally, we write nested things with two more
-spaces of indentation. If your vscode insists on using tabs or four spaces
-consider changing the following settings:
-- `editor.tabSize` to 2, or change `editor.indentSize` to 2.
-- `editor.insertSpaces` to true
-- `editor.detectIndentation` to false
+spaces of indentation. If you have installed the Toit extension, then
+vscode should already do this for you.
```

Comment:
I think it is "Visual Studio Code" or "VS Code", but we probably use vscode in more places.

Reply:
Will change all references in a separate PR.